cpp/src/io/json/nested_json_gpu.cu

```diff
@@ -468,256 +471,411 @@ auto get_transition_table()
 auto get_translation_table()
 {
   std::array<std::array<std::vector<char>, NUM_PDA_SGIDS>, PD_NUM_STATES> pda_tlt;
```

Comment:
on a lighter note, to reduce verbosity, this could be added only inside this function
to get rid of repeated typing of `token_t::`
```

  constexpr auto StructBegin       = token_t::StructBegin;
  constexpr auto StructEnd         = token_t::StructEnd;
  constexpr auto ListBegin         = token_t::ListBegin;
  constexpr auto ListEnd           = token_t::ListEnd;
  constexpr auto StructMemberBegin = token_t::StructMemberBegin;
  constexpr auto StructMemberEnd   = token_t::StructMemberEnd;
  constexpr auto FieldNameBegin    = token_t::FieldNameBegin;
  constexpr auto FieldNameEnd      = token_t::FieldNameEnd;
  constexpr auto StringBegin       = token_t::StringBegin;
  constexpr auto StringEnd         = token_t::StringEnd;
  constexpr auto ValueBegin        = token_t::ValueBegin;
  constexpr auto ValueEnd          = token_t::ValueEnd;
  constexpr auto ErrorBegin        = token_t::ErrorBegin;
```
and follow similar column like arrangement above. 
only issue is that, some symbol groups have multiple outputs, so, indentation will not be regular. 
so, you could wrap around `clang-format off`and `clang-format on` to keep the code's custom alignment (for eg. 5 entries per row)

Reply:
@bdice for suggestions

Reply:
Thanks, I've shortened the names. 

Is there a way to allow people to automatically reformat after they modified the translation table? Because we do want to keep in mind that this is a living piece of code. We pursued this approach because it is flexible and can be adapted to meet current and future feature requests. That means the translation table will be subject to change. I would like to keep barrier to contributions low and avoid people having to fiddle with manually inserting tabs and whitespaces. That's we commented the table to the best of our abilities.